packages/@aws-cdk/aws-rds/test/serverless-cluster.test.ts

```diff
@@ -413,6 +413,22 @@ describe('serverless cluster', () => {
     expect(() => cluster.addRotationSingleUser()).toThrow(/A single user rotation was already added to this cluster/);
 
 
+  });
+
+  test('throws when trying to add rotation to a serverless cluster without vpc', () => {
```

Comment:
Feels like we're missing a similar test for `addRotationMultiUser()`.

Reply:
`addRotationMultiUser()` is not tested at all. I added a test to check it throws but I can't test the rest of the behavior because I don't really what it does.